### DISCUSSION TITLE
6+ months in the sponsorship program waitlist

### DISCUSSION THREAD

**Discussion by elsampsa** [community/community#82398] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a company that wants to sponsor my project, so I requested to be in the github sponsorship program.

Now its been 6+ months in the waitlist - and no reply or admittance from the sponsorship program.  No communication, nothing.

Way to go Microsoft. :)

**Comment by DHayes10** [comment]
Hi @elsampsa 

We appreciate you reaching out. It appears your country of residence is listed as Andorra. Currently, our program is not available in that region as of yet. Please visit https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#supported-regions-for-github-sponsors for more information. 

Thank you, 
GitHub Sponsors

**Reply by elsampsa** [reply]
Thanks for the reply.

But now I have recently moved (back) to Finland & updated my github profile and my sponsorship program application accordingly.  

Again, it seems that it has been left to queue indefinitely, without any notice of problems / errors / success.

What should I do?
